etc/testing/circle/workloads/pulumi/aws/app.go

```diff
@@ -190,13 +190,13 @@ func DeployApp(ctx *pulumi.Context, k8sProvider *kubernetes.Provider, saRole *ia
 				"tag": pulumi.String(pachdImageTag),
 			},
 			"storage": pulumi.Map{
-				"backend": pulumi.String("AMAZON"),
+				"backend":      pulumi.String("AMAZON"),
+				"gocdkEnabled": pulumi.Bool(true),
+				"storageURL":   pulumi.Sprintf("s3://%s", bucket.Bucket),
 				"amazon": pulumi.Map{
-					"gocdkEnabled": pulumi.Bool(true),
-					"storageURL":   pulumi.Sprintf("s3://%s", bucket.Bucket),
-					"region":       pulumi.String("us-west-2"),
-					"id":           pulumi.String(awsSAkey),
-					"secret":       pulumi.String(awsSAsecret),
+					"region": pulumi.String("us-west-2"),
```

Comment:
FWIW you can make the region a url query parameter on the storageURL `s3://bucket?region=us-west-2`

Reply:
Done.